### PR TITLE
Update landing-page.scss

### PR DESCRIPTION
**Pull request by HimajPatil**
Background gradient updated, now more clean black-aqua gradient

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by saurabh100ni** [comment]
Hi @HimajPatil  ,
Thanks for your PR! I encourage you to share this in the Website Meeting tomorrow at 6:30 PM IST. Share your screen, and walk us through it. It's cool to take and share feedback. Check the [Doc](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit) here to include this PR.

You're welcome at the meeting

Check to [community calendar](https://layer5.io/community/calendar) for more details of meetings at Layers.

**Comment by sudhanshutech** [comment]
@HimajPatil why this change?

File: _sass/landing-page.scss [hunk 1]
```diff
@@ -469,8 +469,9 @@
 .getting-started {
   color: #ffffff;
   text-align: center;
-  background: var(--background-grey);
+  // background: var(--background-grey);
   // background: linear-gradient(to top, $menu-color -80%, $primary-brand-color 80%) no-repeat #a05fb7;
+  background: linear-gradient(to right bottom, #000000,#000000, #00483c);
```
Comment (sudhanshutech) [inline _sass/landing-page.scss:474]: we don't define colors like this, it is defined like variables in root.scss and then used

**Comment by sudhanshutech** [comment]
Please open a issue describing the need, then will get on it. Thanks, if you need we will assign you to some good issues.

**Pull request closed without merging**